requirements_dev.txt

```diff
@@ -5,5 +5,3 @@ types-requests
 pre-commit==3.5.0
 tox==4.11.3
 wheel==0.41.3
-
--e .
```

Comment:
The point with this was to install the local directory package in development mode. The screenshot example in the PR description doesn't seem to be executed from the project directory.

Reply:
Original error during container build: [https://pastebin.com/raw/qxTZmgHH](https://pastebin.com/raw/qxTZmgHH)

I didn't find a way to make "-e ." work inside the development txt files.

Reply:
I'd refactor the requirements files and restore the intention of this file. Eg add a new requirements file to create the docker image.

The docker image also needs to install the local project. That seems to be missing now.

Reply:
> The docker image also needs to install the local project. That seems to be missing now.

Sorry for maybe stupid question, but why does the docker image need it? It builds correctly and I can run aioshelly code in it. What am I missing?

Reply:
The project should normally be installed in development mode. This is what I get when I rebuild the container now on main and run in terminal:

```
vscode ➜ /workspaces/aioshelly (main) $ pip show aioshelly
WARNING: Package(s) not found: aioshelly
vscode ➜ /workspaces/aioshelly (main) $ pip install -e .
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///workspaces/aioshelly
  Preparing metadata (setup.py) ... done
Requirement already satisfied: bluetooth-data-tools>=0.3.0 in /usr/local/lib/python3.9/site-packages (from aioshelly==6.0.0) (0.3.1)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.9/site-packages (from aioshelly==6.0.0) (3.9.0)
Requirement already satisfied: orjson>=3.8.1 in /usr/local/lib/python3.9/site-packages (from aioshelly==6.0.0) (3.9.10)
ERROR: Package 'aioshelly' requires a different Python: 3.9.17 not in '>=3.10'

[notice] A new release of pip is available: 23.0.1 -> 23.3.1
[notice] To update, run: pip install --upgrade pip
vscode ➜ /workspaces/aioshelly (main) $
```

Reply:
Thanks for clarification